src/components/modals/UpdateFirmware/Disclaimer.js

```diff
@@ -0,0 +1,201 @@
+// @flow
+/* eslint react/jsx-no-literals: 0 */
+
+import React, { PureComponent, Fragment } from 'react'
+import { translate, Trans } from 'react-i18next'
+import styled from 'styled-components'
+import ReactMarkdown from 'react-markdown'
+
+import type { T } from 'types/common'
+
+import Modal, { ModalBody, ModalFooter, ModalTitle, ModalContent } from 'components/base/Modal'
+import Text from 'components/base/Text'
+import Box from 'components/base/Box'
+import Button from 'components/base/Button'
+import GrowScroll from 'components/base/GrowScroll'
+import GradientBox from 'components/GradientBox'
+
+import { getCleanVersion } from 'components/ManagerPage/FirmwareUpdate'
+
+import type { ModalStatus } from 'components/ManagerPage/FirmwareUpdate'
+
+const Notes = styled(Box).attrs({
```

Comment:
I think we can re-use the style from release notes modal

Reply:
OK I'll change this